src/main/java/Engineer/Engineer.java

```diff
@@ -0,0 +1,93 @@
+package engineer;
+import java.util.Scanner;
+import engineer.exception.EngineerException;
+import engineer.task.*;
+import engineer.command.*;
+
+public class Engineer {
+
+    static Scanner scanner = new Scanner(System.in);
+    static TaskManager myTaskManager = new TaskManager();
+    static ChatBotManager myChatBotManager = new ChatBotManager();
+    static TextDivider myTextDivider = new TextDivider();
+
+    public static void printResponse() throws EngineerException{
+        boolean isKeepAsking = true;
```

Comment:
Think of the name of a boolean variable as a question. I am not quite sure if "is keep asking?" a good name in this case? 
Maybe something like `isRunning` or `shouldContinue` would be more grammartically sound?